package.json

```diff
@@ -6,16 +6,22 @@
     "eject": "expo eject",
```

Comment:
Make sure to run `npx sort-package-json` to have everything nice and sorted

Reply:
done